pallas-validate/src/phase_one/shelley_ma.rs

```diff
@@ -279,13 +283,14 @@ fn check_metadata(tx_body: &TransactionBody, mtx: &MintedTx) -> ValidationResult
 }
 
 fn check_witnesses(
-    tx_body: &TransactionBody,
+    tx_body: &KeepRaw<'_, TransactionBody>,
     tx_wits: &MintedWitnessSet,
     utxos: &UTxOs,
 ) -> ValidationResult {
     let vk_wits: &mut Vec<(bool, VKeyWitness)> =
         &mut mk_alonzo_vk_wits_check_list(&tx_wits.vkeywitness, ShelleyMA(MissingVKWitness))?;
-    let tx_hash: &Vec<u8> = &Vec::from(tx_body.compute_hash().as_ref());
+    let tx_hash: &Vec<u8> = &Vec::from(tx_body.original_hash().as_ref());
+    println!("Tx hash: {:02x?}", tx_hash.clone());
```

Comment:
is this required?

Reply:
The second line is not needed, thanks. It was there for debugging.